pipeline/lib/pipeline-stack.ts

```diff
@@ -24,17 +24,19 @@ export class PipelineStack extends cdk.Stack {
       encryption: BucketEncryption.S3_MANAGED
     });
 
-    // Source (https://docs.aws.amazon.com/cdk/api/latest/docs/aws-codepipeline-actions-readme.html)
+    // Source
+    const gitRepo = "GIT_REPO_NAME"
+    const gitOwner = "GIT_REPO_OWNER"
+    const gitBranch = "GIT_BRANCH"
```

Comment:
I still doubt if I'd get these values from something like SSM, so code doesn't need to be amended for each user. At the same time, this is a demo repo, we should draw a line somewhere :)

Reply:
I wouldn't use the SSM here; otherwise, we will need to explain how to set up the ssm. But yes, I would use ssm for my own project.

Reply:
@jbernalvallejo 
I have submitted a commit with the changes and I believe that it also reflects this PR. Let me know if you can see it.